query/common/src/main/java/org/codice/ddf/admin/common/fields/base/BaseObjectField.java

```diff
@@ -40,9 +41,13 @@ public BaseObjectField(String fieldName, String fieldTypeName, String descriptio
     public Map<String, Object> getValue() {
         Map<String, Object> values = new HashMap<>();
 
-        for(Field field : getFields()) {
-            if(!(field instanceof FunctionField)) {
-                values.put(field.fieldName(), field.getValue());
+        for (Field field : getFields()) {
+            if (!(field instanceof FunctionField)) {
+                if (field.fieldName().equals(PasswordField.DEFAULT_FIELD_NAME) && ((PasswordField) field).isInternalProcess()){
```

Comment:
A `PasswordField` is currently only create from a `CredentialField` which always has the filed name of `password` but I see how you can now create a `PasswordField` without having to go through the `CredentialField` and give it a different filed name. 👍  Will change that.